sos/policies/__init__.py

```diff
@@ -1353,4 +1383,53 @@ def upload_ftp(self, url=None, directory=None, user=None, password=None):
         except IOError:
             raise Exception("could not open archive file")
 
+    def set_sos_prefix(self):
```

Comment:
Is this method used anywhere? I can't find it..

Reply:
I _think_ that's a leftover from the previous design for executing within a container. In the current design, that should be handled by `Policy.format_container_command()`.

I'll double check on some containerized hosts this week.